sdk/communication/Azure.Communication.CallAutomation/src/EventHandler/EventAwaiterOngoing.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Communication.CallAutomation
+{
+    internal class EventAwaiterOngoing<TEvent> where TEvent : CallAutomationEventBase
+    {
+        private string _callConnectionId;
+        private Action<TEvent> _eventHandler;
+
+        internal Action<object, CallAutomationEventArgs> OnEventRecieved => OnEventsReceived;
```

Comment:
typo:
```suggestion
        internal Action<object, CallAutomationEventArgs> OnEventReceived => OnEventsReceived;
```
(and update references)